privacy-protections/index.html

```diff
@@ -16,6 +16,7 @@ <h1>Privacy Protections Tests</h1>
         <li><a href='https://privacy-test-pages.glitch.me/privacy-protections/storage-blocking/'>Storage blocking</a></li>
         <li><a href='https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/'>Referrer trimming</a></li>
         <li><a href='http://privacy-test-pages.glitch.me/privacy-protections/https-upgrades/'>HTTPS upgrades</a></li>
+        <li><a href='http://privacy-test-pages.glitch.me/privacy-protections/click-to-load/'>Facebook click to load</a></li>
```

Comment:
Can you please also add this page to the main index.html file? Sorry, it's a bit confusing that we have two 🤦‍♂️

Reply:
yeah totally forgot that one!